CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 
 ### Fixed
 - #1976 - Fixed failing Remote Assets and tests dependent on mock server on JDK 11
+- Fixed the Shared and Global icons that are not appearing in edit bar when the dialog is edited and saved and page refreshes due to Edit Config Listener ( Shared Component Properties )
```

Comment:
please add `#1982 - ` to the front of this line

Reply:
@HitmanInWis just a FYI - Github has the Suggestions feature that lets you "suggest" a change that can be accepted in Github.com -- helps reduce the back and forth on minor changes

![image](https://user-images.githubusercontent.com/1451868/60893648-4c5f7e00-a22f-11e9-865c-5d4d32bbfac1.png)

Reply:
```suggestion
#1982 - Fixed the Shared and Global icons that are not appearing in edit bar when the dialog is edited and saved and page refreshes due to Edit Config Listener ( Shared Component Properties )
```

Reply:
Fancy! 👍

Reply:
> please add `#1982 - ` to the front of this line

Updated the Changelog file the PR number. Thanks!